lighthouse-core/report/html/templates.html

```diff
@@ -574,6 +574,218 @@
   </a>
 </template>
 
+<!-- Lighthouse PWA badge gauge -->
+<template id="tmpl-lh-gauge--pwa">
+  <style>
+    .lh-gauge--pwa {
+      width: calc(3 * var(--header-font-size));
+      height: calc(3 * var(--header-font-size));
+      display: none;
+    }
+
+    .lh-badged-na .lh-gauge--pwa--na {
```

Comment:
these are all just lining up the `lh-badged-*` with `lh-gauge--pwa--*` right? since the intention is all identical and we would ideally replace something simpler maybe it does make sense to combine them?